_docs/tasks/traffic-management/egress.md

```diff
@@ -249,8 +249,11 @@ cloud provider specific knowledge and configuration.
    ```
 
 
-## What's next
+## Further reading
 
 * Read more about [egress rules]({{home}}/docs/concepts/traffic-management/rules-configuration.html#egress-rules).
 
-* Learn how to use Istio's [request routing](./request-routing.html) features.
+* Learn how to use setup
+  [timeouts]({{home}}/docs/reference/config/traffic-rules/routing-rules.html#httptimeout),
+  [retries]({{home}}/docs/reference/config/traffic-rules/routing-rules.html#httpretry),
```

Comment:
kill the `,` at the end

Reply:
Yeah, thats a long argument.. Some say to use the comma before and. @Rachael-Graham whats the English's take on this? [pun intended :P]

Reply:
Haha--I know that at least for IBM style you use the oxford comma. So "... timeouts, retries, and circuit breakers ..."